lib/lte_link_control/lte_lc.c

```diff
@@ -175,10 +175,10 @@ static const char *const system_mode_params[] = {
 static struct k_sem link;
 
 #if defined(CONFIG_LTE_PDP_CMD) && defined(CONFIG_LTE_PDP_CONTEXT)
-static const char cgdcont[] = "AT+CGDCONT="CONFIG_LTE_PDP_CONTEXT;
+static char cgdcont[120] = "AT+CGDCONT="CONFIG_LTE_PDP_CONTEXT;
```

Comment:
Are those buffer sizes reasonable defaults @jtguggedal? Not sure how long apn/username/password can be. Probably those sizes should be Kconfig configurable?

Reply:
APN is limited to 100 characters. I couldn't find a limit for username/password.

Reply:
I can't find specified limits in the AT spec either, so I guess these are reasonable defaults, but making them Kconfigurable would be a nice improvement. Then the defaults can be lowered a bit.